torch/_inductor/config.py

```diff
@@ -156,11 +162,10 @@ class triton:
     # limit tiling dimensions
     max_tiles = 2
 
-    # use triton.autotune?
+    # use triton.autotune for pointwise ops with complex layouts
+    # this should only be disabled for debugging/testing
```

Comment:
We should probably change the name of this to `autotune_pointwise` or something.  Maybe in a separate diff.

Reply:
will do